catalog/src/main/java/it/eng/catalog/rest/api/OfferAPIController.java

```diff
@@ -12,11 +12,11 @@
 import it.eng.catalog.model.Offer;
 import it.eng.catalog.serializer.Serializer;
 import it.eng.catalog.service.CatalogService;
-import lombok.extern.java.Log;
+import lombok.extern.slf4j.Slf4j;
 
 @RestController
 @RequestMapping(consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE, path = "/api/offer")
```

Comment:
Use plural instead of singular: e.g. offer -> offers.

Also, we agreed that we would integrate API versioning, so v1 should be includedalso in path: /api/v1/offers